commands/status/process_test.go

```diff
@@ -238,6 +238,10 @@ var testCasesExtractBranch = []struct {
 		[]byte("## Initial commit on master"),
```

Comment:
I'm not sure when this message gets printed. Is this for an older version of `git`? Maybe we should put a comment to describe how to replicate it in a real system.

Reply:
I wonder if perhaps this is in fact the same message and it just changed text in more recent versions of git. Perhaps a search through the git codebase could tell us that.

Reply:
Ah yeah, looks like it was changed in https://github.com/git/git/commit/4ddb1354e8d5daf5671d3d451a67d2d1e82d9b49